docs/source/science/ESA_CCI/ESA_CCI_V4.ipynb

```diff
@@ -17,52 +17,39 @@
     "tags": []
```

Comment:
<p>This is really picky, but in other notebooks, we've had the "Run This Notebook" section above the "About the Dataset" section. I don't think they necessarily have to be switched, but if we wanted to keep it more consistent with other tutorials, they could be.</p><p> </p><p>Another section we typically add is "Additional Resources" and try to link more information about the data (dataset landing pages, user guides, information about how to use a specific tool, etc.). You can add this if you'd like, but I also wouldn't mind going back in and adding it! However, after your paragraph under "About the Dataset", I would at least add a link with where that information came from for the users.</p><p> </p><p>Also -- in your first sentence under "About the Dataset", I would put "(AGB)" next to "above-ground biomass". I only recommend this because because you spell it out up here, but just use "AGB" below from what I can tell. It's just another thing we've been doing in tutorials so users know the acronyms 🙂</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/335/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>